src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -118,6 +118,8 @@ public enum SlackDistributionFailureBehavior {
 
     public static final boolean WRITE_REFERENCE_TERMINALS_DEFAULT_VALUE = true;
 
+    protected static final double GENERATOR_VOLTAGE_CONTROL_MIN_NOMINAL_VOLTAGE_DEFAULT_VALUE = 0;
+
```

Comment:
@annetill why is the default 0 ? (and not -1 for automatic ? )

Reply:
why -1 would be better that 0 ?

Reply:
as discussed in other places, to allow no limit (why not allowing this ?  :-) )